scripts/test-upgrade.sh

```diff
@@ -105,6 +105,25 @@ cd ${OA_DIR}/playbooks
 ansible galera_all[0] -m shell -a "mysql --verbose -e \"${VLAN_FLAT}\" neutron"
 ansible galera_all[0] -m shell -a "mysql --verbose -e \"${VXLAN}\" neutron"
 
+# CEPH CHANGES
+# These are the instructions to give if there is a failure message on ceph-mon.yml
```

Comment:
```
root@liberty-aio:/opt/rpc-openstack/rpcd/playbooks# ansible monss --list-hosts | wc -w
No hosts matched
0
root@liberty-aio:/opt/rpc-openstack/rpcd/playbooks#
```

No issues, just thought we could avoid the extra ansible call by saving the results.  I'm OK either way.